src/wwElement.vue

```diff
@@ -300,7 +299,8 @@ export default {
             this.resizeObserver.observe(this.$el, { box: 'device-pixel-content-box' });
         },
         checkIsOpen() {
-            if (this.isEditing && this.content.isOpen) this.handleOpening(true);
+            if (!this.isEditing) return;
+            this.handleOpening(this.wwEditorState.sidepanelContent.openInEditor);
```

Comment:
Meême remarque que Alexis sur le wwEditorState qui n'existe pas sur le site publié.
Tu peux pas juste strip ça ?

Reply:
Effectivement ici, je peu strip la ligne 303, mais je vais quand même devoir laisser la vérification, car la fonction sera appelée, même sur le site publié

Reply:
Vu que je dois laisser la fonction (car le select va l'appeler aussi sur le site publié), je trouve ça plus clair de laisser faire la logique du `if (!this.isEditing) return;`

<img width="619" alt="CleanShot 2022-11-04 at 10 57 19@2x" src="https://user-images.githubusercontent.com/39409397/199945253-ea838f4c-473b-418c-824f-236c5de8e06f.png">

Je trouve le 1er plus propre que le 2eme

Reply:
Tu peux utiliser le 2e, on le fait à d'autres endroits, c'est pas très grave
Et comme ça on est sûr à 100% que ça ne cassera pas.